### PR TITLE
Added  scrum,buildtool, linux and git commands

### PR DESCRIPTION
**Pull request by kulkarni90**

### CODE REVIEW

File: phani-2/buildtool.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+
```
Comment (david1412) [inline phani-2/buildtool.txt]: Remove the space

**Review by david1412 — COMMENTED** [review]
please check the comment

File: phani-2/buildtool.txt (hunk 1 above)
Comment (kulkarni90) [reply phani-2/buildtool.txt]: Ok anna will correct it next time

**Review by kulkarni90 — COMMENTED** [review]
Removed space as per the comments

File: phani-2/linux-commands-2.txt [hunk 2]
```diff
@@ -0,0 +1,27 @@
+The most frequently used Linux commands that are used while working in DevOps.
+
+1.sudo=This command executes only that command with root/ superuser privileges.
```
Comment (pradeepm2406) [inline phani-2/linux-commands-2.txt]: Please leave space after the serial number  (ex: 1. sudo  and not 1.sudo) for all the points

**Review by pradeepm2406 — COMMENTED** [review]
Please edit linux commands file by keeping space between serial number and command.

**Review by prudhvidhargudimalla — APPROVED** [review]
no repeated commands compare to task 1 and neat presentation all over its good

**Review by kulkarni90 — COMMENTED** [review]
Corrected  the space after sr.number. thanks for suggetion anna .

**Review by nithindeva — APPROVED** [review]
NIce info

**Review by pradeepm2406 — APPROVED** [review]
All looks good

**Review by nithindeva — APPROVED** [review]

**Pull request merged**